pkg/apis/servicecatalog/validation/broker_test.go

```diff
@@ -503,12 +543,12 @@ func TestValidateClusterServiceBroker(t *testing.T) {
 						RelistBehavior: servicecatalog.ServiceBrokerRelistBehaviorManual,
 						CatalogRestrictions: &servicecatalog.CatalogRestrictions{
 							ServiceClass: []string{
-								"name==barfoobar",
-								"externalName in (barfoobar, batbazboof, batwizzbang)",
+								"name=barfoobar",
```

Comment:
any reason to this changed from name==barfoobar to name=barfoobar?  = and == are synonymous right?

Reply:
There were no tests with =, even though = is a valid conditional. I changed it so that we have at least one test with a =. We now need this test because we're now splitting the filter condition to extract the property.